src/core/border_router/routing_manager.cpp

```diff
@@ -3577,6 +3595,15 @@ void RoutingManager::PdPrefixManager::SetEnabled(bool aEnabled)
     return;
 }
 
+void RoutingManager::ProcessPlatfromGeneratedRa(const uint8_t *aRouterAdvert, uint16_t aLength)
+{
+    mPdPrefixManager.ProcessPlatformGeneratedRa(aRouterAdvert, aLength);
+    // Collect the number of platform received RA when DHCPv6 PD enabled.
+    mNumPlatformRaReceived++;
```

Comment:
Remove this comments and the one below?

It is clear from code itself what is being done and the comment is just explaining the line of code that follows it.

When adding comments, I generally consider whether they genuinely aid understanding by clarifying some aspect that is not readily apparent from the code itself. It is always better to let the code convey the logic whenever possible.